amazon/no-nap/Dockerfile

```diff
@@ -72,6 +72,15 @@ RUN set -ex \
 RUN ln -sf /dev/stdout /var/log/nginx-controller/agent.log \
   && ln -sf /dev/stderr /var/log/nginx/error.log
 
+RUN chown -R nginx:nginx /var/run/nginx-controller/ \
```

Comment:
please add comment explaining why this is being done

Reply:
done